qual/src/main/java/com/virtuslab/qual/gitmachete/frontend/resourcebundles/LooseHtml.java

```diff
@@ -0,0 +1,17 @@
+package com.virtuslab.qual.gitmachete.frontend.resourcebundles;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.checkerframework.common.subtyping.qual.Unqualified;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+import com.virtuslab.qual.internal.SubtypingTop;
+
+
+@SubtypeOf(Unqualified.class)
```

Comment:
I probably haven't though of using `Unqualified` for the top-level type of all these annotations... if it works as expected for your new `@...Html` annotations, then considered removing `com.virtuslab.qual.internal.SubtypingTop`.

TBD if `com.virtuslab.qual.internal.SubtypingBottom` can be removed as well... but this might be harder

Reply:
Oh! judging by the javadoc for `org.checkerframework.common.subtyping.qual.Unqualified` and `org.checkerframework.common.subtyping.qual.Bottom`... our own `SubtypingTop` and `SubtypingBottom` should be retained & used everywhere :/

Reply:
it doesn't work as it should with `Unqualified`

Reply:
with our `SubtypingTop` when I test all this flow, check framework does not check if the given method returns exact e.g. `@...Html` but check for `SubtypingTop`.  I need to read more doc about it
<img width="1004" alt="Screenshot 2022-06-08 at 12 38 18" src="https://user-images.githubusercontent.com/33667003/172596652-a9f5f045-6c07-4977-8cb3-1305c15756de.png">

Reply:
Hmmm I think this is expected 🤔 as in, a plain "unsubtyped" String will indeed be seen as `@SubtypingTop String`

Reply:
...anyway... lemme push a few changes onto a separate PR... I think they might be useful for you

Reply:
a okey in this situation format method return plain String but method `newAnnotation` required `@NonHtml`

Reply:
Awww crap I see the problem :ups:

Reply:
Shiiiiiit that might be difficult to work around ☹️

Reply:
As in, when `format` is NOT involved, `SubtypingChecker` will work as expected... but how to retain the `@...Html` from `format`'s param into `format`'s return type? 😕

Reply:
Not Good ™️

Unless you find some simple solution here... I have one more idea (https://checkerframework.org/manual/#tainting-checker)... and if this doesn't work, I'd simply resort to documenting in CONTRIBUTING.md what we know about the format accepted by each method... and probably call it a day

Reply:
I will try (before the refactor of the rest of the code)